### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by mettix** [chadmayfield/pihole-blocklists#4]
I have added the file in the adlist, it shows 1800000 domains blocked, but it is not working.

pronhub is going through when I check the logs.. its like it is not  using the list?

Any idea?

**Comment by chadmayfield** [comment]
Hi, are you using the subdomain www when it isn't blocked? There is an issue with that (#3) that I have not fixed yet since I don't have time.

**Comment by mettix** [comment]
I opened them from google, I'll try to write a batch script to duplicate the entries. But it is weird, when I use the /admin/queryads.php page to search for pornhub whatsoever I don't get any matches (or is that normal?)

**Comment by chadmayfield** [comment]
I'm assuming that when opening from google.com it is linked to the www subdomain so it will not work... which is a [known issue](https://github.com/chadmayfield/my-pihole-blocklists/issues/3) (which at some point I will redo my lists to take into account those unblocked domains).  For right now you can get on your pi-hole and run nslookup www.domain.com to see to where it is routing.  Is the machine that you are using using a different DNS?

Just remember there is [already an issue](https://github.com/chadmayfield/my-pihole-blocklists/issues/3) that will miss some subdomains (in this case the www) that needs to be fixed.

**Comment by chadmayfield** [comment]
Closed as duplicate of issue #3
